src/block/count-up/edit.js

```diff
@@ -60,7 +62,21 @@ const Edit = props => {
 		<Fragment>
 
 			<InspectorTabs />
+			<InspectorBlockControls>
+				<PanelAdvancedSettings
+					title={ __( 'Speed', i18n ) }
```

Comment:
It might be better if the title was "Counter" instead of "Speed" so that it would still be good if our panel were to contain other options aside from the duration.